pkg/apis/experimental/types.go

```diff
@@ -495,11 +495,14 @@ type IngressList struct {
 // IngressSpec describes the Ingress the user wishes to exist.
 type IngressSpec struct {
 	// A default backend capable of servicing requests that don't match any
-	// IngressRule. It is optional to allow the loadbalancer controller or
+	// IngressRule. Either an IngressBacked or a set of IngressRules must be
```

Comment:
s/Backed/Backend -- actually this should use json names.

This comment makes it sound like an XOR.  Maybe something like
"At least one of 'backend' or 'rules' must be specified.  This field is optional to allow..."